### PR TITLE
Allow purging multiple projects at once

### PR DESCRIPTION
**Pull request by dbnicholson**
https://phabricator.endlessm.com/T4329

### CODE REVIEW

File: src/backend/bs_admin [hunk 1]
```diff
@@ -748,13 +750,17 @@ while (@ARGV) {
       }
     }
 
+    die("ERROR: no projects specified\n") if @projects < 1;
```
Comment (freeman-endlessm) [inline src/backend/bs_admin]: Perl aray length is $#, no? As in: `$#projects`? 

```
freeman@x250:~$ perl
@p=[];
if (@p < 1){ print "blah";}
freeman@x250:~$ 
freeman@x250:~$ perl
@p=[];
if ($#p < 1){ print "blah";}
blah
```

Comment (dbnicholson) [inline src/backend/bs_admin]: I think I'll just use `scalar` as `$#` seems a little weird.

```
$ re.pl
$ my @foo;

$ @foo < 1;
1
$ $#foo
-1
$ scalar(@foo)
0
$ @foo;

$ scalar(@foo) < 1;
1
```

Comment (freeman-endlessm) [inline src/backend/bs_admin]: Sounds good to me

**Comment by dbnicholson** [comment]
@freeman-endlessm CR ready again with that `scalar` fix.

**Comment by freeman-endlessm** [comment]
LGTM

**Pull request merged**